homeassistant/components/husqvarna_automower/sensor.py

```diff
@@ -141,6 +300,22 @@ class AutomowerSensorEntityDescription(SensorEntityDescription):
         device_class=SensorDeviceClass.TIMESTAMP,
         value_fn=lambda data: dt_util.as_local(data.planner.next_start_datetime),
     ),
+    AutomowerSensorEntityDescription(
+        key="error",
+        translation_key="error",
+        device_class=SensorDeviceClass.ENUM,
+        value_fn=lambda data: (
+            "no_error" if data.mower.error_key is None else data.mower.error_key
+        ),
+        options=ERROR_KEY_LIST,
+    ),
+    AutomowerSensorEntityDescription(
+        key="restricted_reason",
+        translation_key="restricted_reason",
+        device_class=SensorDeviceClass.ENUM,
+        options=RESTRICTED_REASONS,
```

Comment:
```suggestion
        options=[option.value.lower() for option in list(RestrictedReasons)],
```

Reply:
I already had it like this, but I changed it. See comment [here](https://github.com/home-assistant/core/pull/113165#discussion_r1545754481). Should I change it back?